tests/comments/test_views.py

```diff
@@ -171,8 +171,10 @@ def test_reply_for_reply(app: DjangoTestApp):
 
     app.set_user(user)
     form = app.get(comment.content_object.get_absolute_url()).forms['reply-form']
-    form['is_public'] = True
-    form['body'] = "Test reply"
+    form.fields.get('is_public')[0].value = True
+    form.fields.get('body')[0].value = "Test reply"
+    form.fields.get('is_public')[1].value = True
+    form.fields.get('body')[1].value = "Test reply"
```

Comment:
Wait, but you're using `.get()` to access `is_public` or `body` what if form.fields do not have them (based that you are not indexing strictly with `[]` and using `get()` instead) ? None is not subscriptable